issue-with-transitions.json

```diff
@@ -0,0 +1,116 @@
+{
+    "$id": "https://example.com/schemas/issue",
+    "$schema": "http://json-schema.org/draft-07/hyper-schema#",
+    "base": "https://example.com/api/",
+    "title": "Issue Tracker",
+    "definitions": {
+        "issue": {
+            "type": "object",
+            "required": ["title", "status"],
+            "properties": {
+                "id": {
+                    "description": "Server-assigned unique identifier; will always be present once assigned during creation",
+                    "type": "integer",
+                    "minimum": 1,
+                    "readOnly": true
+                },
+                "title": {"type": "string"},
+                "status": {
+                    "type": "string",
+                    "enum": ["Open", "Closed"],
+                    "default": "Open"
+                },
+                "description": {"type": "string"}
+            },
+            "links": [
+                {
+                    "description": "",
+                    "rel": "self",
+                    "href": "issue/{id}",
+                    "templateRequired": ["id"],
+                    "targetSchema": {"$ref": "#/definitions/issue"},
+                    "targetHints": {
+                        "allow": ["GET", "PUT"]
+                    },
+                    "x-dataTransitions": {
+                        "open": {
+                            "mediaType": "application/merge-patch+json",
+                            "patch": [
```

Comment:
Ok so you are already going offbook. This should be POST. There is a `transition` link that is a PUT we can include.

Reply:
That has nothing to do with hyper-schema, that's just what I thought the operation was.  It's just updating a field in the representation, the update is idempotent, and the URI is known.  Why would it be a POST?  I can, of course, rework the schema for that, I'm just curious.

Reply:
So all fair points. The idea of modeling an existing api is to remove any debate around design and focus entirely on the description language. So there should be no ambiguity. Sorry if I didn't make this clear. The implementation for this particular operation is post. I can link to the specific code if that helps.

Reply:
Oh, I see.  I thought you wanted a more RESTful version of something that was already an RPC-ish spec.

Hyper-Schema doesn't fit RPC well, so this might actually not work out as an example.  It can be done if each call has its own URI, but it doesn't end up being very interesting.  You put the request schema in `submissionSchema` and that's about it.  If the POST response with an updated representation you can describe that with `targetSchema`, but if not then there's no direct response description (because it should link to its own schema).